dotnet/src/SemanticKernel.Abstractions/AI/ChatClient/ChatMessageExtensions.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Extensions.AI;
+using Microsoft.SemanticKernel.ChatCompletion;
+using System;
+
+namespace Microsoft.SemanticKernel.AI.ChatCompletion;
+
+internal static class ChatMessageExtensions
+{
+    /// <summary>Converts a <see cref="ChatMessage"/> to a <see cref="ChatMessageContent"/>.</summary>
+    /// <remarks>This conversion should not be necessary once SK eventually adopts the shared content types.</remarks>
+    internal static ChatMessageContent ToChatMessageContent(this ChatMessage message, ChatResponse? response = null)
```

Comment:
This is just moving around the existing support, or is there new functionality here?

Reply:
Moving around for the actual `sponsor` class names.